.github/workflows/checklist_actions.yml

```diff
@@ -9,31 +9,28 @@ jobs:
     if: ${{ github.event.issue.pull_request && startsWith(github.event.comment.body, 'add_reformatting_checklist_') }}
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - run: |
-          echo A comment on PR $NUMBER
-          echo The body of comment $ID is "$BODY"
-        env:
-          NUMBER: ${{ github.event.issue.number }}  
-          BODY: ${{ github.event.comment.body }}
-          ID: ${{ github.event.comment.id }}
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
I suggest you checkout to ivy like this -:
```
 uses: actions/checkout@v2
        with:
          path: ivy
          persist-credentials: false
```

and put `reformat_checklist.md` somewhere inside `automation tools`